howto/anbox/auth.md

```diff
@@ -0,0 +1,108 @@
+(howto-auth)=
+## Configure user permissions
+
+To be able to configure user permissions in Anbox Cloud, you need to first configure OpenFGA.
+
+## Configure OpenFGA
+
+```{important}
+This section is required only for the charmed deployment. For the Anbox Cloud Appliance, you can skip this step and proceed to creating identities.
+```
+Anbox Management Service(AMS) requires a store ID in OpenFGA to manage authorization data. To generate a store ID in OpenFGA, follow the steps in the [OpenFGA documentation](https://openfga.dev/docs/getting-started/create-store) and create a store. 
+
+When you have a store ID, run these commands to configure AMS:
+
+```
+amc config set openfga.api.url '<URL of your OpenFGA instance>'
+amc config set openfga.store.id '<store id of the user created store>'
+```
+
+Optionally, you can also configure the API token if your deployment setup will require authentication for API access to OpenFGA:
+
+    amc config set openfga.api.token '\<api token generated from OpenFGA\>'
+
+When the API URL and store ID are set, AMS starts synchronizing the data with OpenFGA. You might see some authorization failures initially till the synchronization completes.
+
+## Create identities
+
+To create an OIDC identity, run:
+
+    amc auth identity create oidc/test@example.com
+
+To create a TLS identity, run:
+
+    cat client.crt | amc auth identity create tls/test-user
+
+You can view the identities you created by using:  
+
+    amc auth identity ls
+
+The output will look similar to this:
+
+```{terminal}
+`+--------------------+---------------------+----------------------+----------------------+--------------+`  
+`|    ID       | AUTHENTICATION TYPE |      NAME        |                            IDENTIFIER          |    GROUPS    |`  
+`+----------------------+---------------------+------------------+------------------------+--------------+`  
+`| d3kb9pvriueuguose410 | oidc  | test@example.com   | test@example.com    | image_viewer |`  
+`+----------------------+---------------------+-------------------------+----------------------+---------+`  
+`| d3o2u7vriuenvvlfcae0 | tls   | test-user | 9cbb1aced3fb587405c7abaa7c83ec59b4bb9bd567843703884632e77e0aa455 |   |`  
+`+----------------------+---------------------+-----------------------+--------------------+-------------+`
+```
+
+## Create authorization groups with identities
+
+### Create groups
+
+Create a new authorization group:
+
+    amc auth group create developer
+
+To verify details of the created group, use the `show` command:
+
+```{terminal}
+$ amc auth group show developer
+name: developer
+created-at: 2025-10-17 12:06:33 +0530 IST
+updated-at: 2025-10-17 12:06:33 +0530 IST
+identities: []
+permissions: []
+immutable: false
+```
```

Comment:
This now looks a bit weird too?

<img width="1518" height="532" alt="Image" src="https://github.com/user-attachments/assets/992f3e59-05d7-4162-a51a-2d64eac14b99" />